temporalcli/commandsgen/commands.yml

```diff
@@ -3439,6 +3439,160 @@ commands:
           - accepted
         required: true
 
+  - name: temporal workflow start-update-with-start
+    summary: Send an Update and wait for it to be accepted or rejected (Experimental)
+    description: |
+      Send a message to a Workflow Execution to invoke an Update handler, and wait for
+      the update to be accepted or rejected. If the Workflow Execution is not running, 
+      then a new workflow execution is started and the update is sent.
+
+      Experimental.
+
+      ```
+      temporal workflow start-update-with-start \
+        --update-name YourUpdate \
+        --update-input '{"update-key": "update-value"}' \
+        --workflow-id YourWorkflowId \
+        --type YourWorkflowType \
+        --task-queue YourTaskQueue \
+        --input '{"wf-key": "wf-value"}'
```

Comment:
This example doesn't show `--update-wait-for-stage`, but it is a required option.
(`--id-conflict-policy` is also required)

Reply:
Added both to the example.
Added `--wait-for-stage` to the `temporal workflow update start` example as well (also required).